test/system/005-info.bats

```diff
@@ -53,4 +53,8 @@ store.imageStore.number   | 1
 
 }
 
+@test "podman info --storage-opt="" " {
+    run_podman --storage-opt="" info
```

Comment:
This completely blows up on my laptop (rootless, that is; works fine as root of course):
```
Error: kernel does not support overlay fs: 'overlay' is not supported over btrfs at "/home/esm/.local/share/containers/storage/overlay": backing file system is unsupported for this graph driver
```